packages/components/src/components/install-guide/record-instructions/Python.vue

```diff
@@ -1,66 +1,78 @@
+      <section class="recording-method">
+        <h3>
+          <!--TODO make recommended tag dynamic-->
+          <i class="header-icon"><TestsIcon /></i>Tests recording<span class="recommended-badge"
+            >recommended</span
+          >
+        </h3>
+        <p>
+          When you run your {{ testFramework.name }} tests with AppMap enabled, an AppMap will be
+          created for each test.
+        </p>
+        <template class="recording-method" v-if="testFramework.name.toLowerCase() == 'pytest'">
+          Run Pytest tests:
+          <v-code-snippet clipboard-text="pytest" />
+        </template>
+        <p>
+          For more information, visit
+          <a
+            href="https://appmap.io/docs/reference/appmap-python.html#tests-recording"
+            target="_blank"
+            >AppMap docs - Python Tests recording</a
+          >.
+        </p>
+      </section>
     </template>
+    <template v-else> <VTestsPrompt frameworks="pytest or unittest" /> </template><br />
     <template v-if="webFramework">
-      <h3>Requests recording</h3>
-      <p>
-        When your application uses {{ this.webFramework.name }}, and you run your application with
-        AppMap enabled, HTTP server requests recording is enabled. To record requests, first run
-        your application:
-      </p>
-      <p>
-        Start your Django server:
-        <v-code-snippet clipboard-text="python manage.py runserver" />
-      </p>
-      <p>
-        Start your Flask server:
-        <v-code-snippet clipboard-text="flask run" />
-      </p>
-      <p>
-        Interact with your application, through its user interface and/or by making API requests
-        using a tool such as Postman. An AppMap will be created for each HTTP server request that's
-        served by your app.
-      </p>
-      <p>
-        For more information, visit
-        <a
-          href="https://appmap.io/docs/reference/appmap-python.html#requests-recording"
-          target="_blank"
-          >AppMap docs - Python Requests recording</a
-        >.
-      </p>
+      <section class="recording-method">
+        <h3>
+          <i class="header-icon"><RequestsIcon /></i>Requests recording
+        </h3>
+        <p>
+          When your application uses {{ this.webFramework.name }}, and you run your application with
+          AppMap enabled, HTTP server requests recording is enabled. To record requests, first run
+          your application:
+        </p>
+        <p>
+          Start your Django server:
+          <v-code-snippet clipboard-text="python manage.py runserver" />
+        </p>
+        <p>
+          Start your Flask server:
+          <v-code-snippet clipboard-text="flask run" />
+        </p>
+        <p>
+          Interact with your application, through its user interface and/or by making API requests
+          using a tool such as Postman. An AppMap will be created for each HTTP server request
+          that's served by your app.
```

Comment:
Drop the "that's" word, as it makes it sound like they'll get an AppMap for every registered API route. (at least to me!)

Reply:
removed